webpack.config.node.js

```diff
@@ -87,17 +87,17 @@ const webpackConfig = {
 					cacheIdentifier: cacheIdentifier,
 				}
 			},
-			{
-				test: /\.json$/,
-				exclude: /(devdocs[\/\\]components-usage-stats.json)/,
-				loader: 'json-loader'
-			}
 		]
 	},
 	resolve: {
-		extensions: [ '', '.json', '.js', '.jsx' ],
-		root: [ __dirname, path.join( __dirname, 'server' ), path.join( __dirname, 'client' ), path.join( __dirname, 'client', 'extensions' ) ],
-		modulesDirectories: [ 'node_modules' ]
+		modules: [
+			path.join( __dirname, 'server' ),
+			path.join( __dirname, 'client' ),
+			path.join( __dirname, 'client', 'extensions' ),
+			__dirname,
```

Comment:
Unless you have a reason not to, `__dirname` should be first (to have higher priority than `extensions` and such). Not *needed* right now, but it could lead to difficult-to-track bugs down the line.